lib/rules/no-empty.js

```diff
@@ -51,6 +53,7 @@ module.exports = {
 
         return {
             BlockStatement(node) {
+                const range = [node.range[0] + 1, node.range[1] - 1];
```

Comment:
Can we move this into `fix(fixer)`, since it's needed only for the fix, and at this point it isn't clear what the `range` represents.

Reply:
yes, we can move.